packages/dotcom-ui-footer/src/__test__/component.spec.tsx

```diff
@@ -27,6 +27,14 @@ describe('dotcom-ui-footer', () => {
     expect(legalFooter.find('.o-footer__brand-logo')).toExist()
   })
 
+  it('renders a custom class on the footer', () => {
+    dataFixture.customClass = 'foo'
+    const footer = mount(<Footer {...dataFixture} />)
+    const legalFooter = mount(<Footer {...dataFixture} />)
+    expect(footer.find('.o-footer').hasClass('foo')).toBe(true)
+    expect(legalFooter.find('.o-footer').hasClass('foo')).toBe(true)
+  })
```

Comment:
Might be worth adding a test to see what the entire class value looks like when `customClass` is absent, in relation to other comment about `undefined` possibly being included.